lucet-runtime/lucet-runtime-internals/src/instance.rs

```diff
@@ -602,7 +614,7 @@ impl Instance {
             *rip_addr_details = self.module.addr_details(rip_addr as *const c_void)?.clone();
 
             // If the trap table lookup returned unknown, it is a fatal error
-            let unknown_fault = trapcode.ty == TrapCodeType::Unknown;
+            let unknown_fault = trapcode.is_none();
```

Comment:
Not anything about this change in particular - what would cause an unknown trap? A malformed module with incorrect trap tables?

Reply:
yes, if its in the module it means there's either a bug in cranelift or lucetc. it could also be a trap in the host code.